### PR TITLE
feat: add `MsgUpdateCensorship` msg and `Censorship` query of x/foundation API

### PR DESCRIPTION
**Pull request by zemyblue**
## Description

* add `MsgUpdateCensorship` msg api of x/foundation
* add `Censorship` query api of x/foundation
* update script setting for fitting with finschia-sdk v0.48.0 and finschia v2.0.0

related: #100 

## Checklist:

- [x] I have added a relevant changelog to `CHANGELOG.md`.
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by jaeseung-bae — DISMISSED** [review]
LGTM

File: scripts/simapp/template/.simapp/config/genesis.json [hunk 1]
```diff
@@ -281,13 +283,13 @@
               "aliases": []
             },
             {
-              "denom": "LN",
+              "denom": "FNSA",
```
Comment (0Tech) [inline scripts/simapp/template/.simapp/config/genesis.json:286]:
```suggestion
              "denom": "finschia",
```

File: scripts/simapp/template/.simapp/config/genesis.json [hunk 2]
```diff
@@ -281,13 +283,13 @@
...
               "exponent": 6,
               "aliases": []
             }
           ],
-          "base": "LINK",
-          "display": "LN"
+          "base": "FINSCHIA",
+          "display": "FNSA"
```
Comment (0Tech) [inline scripts/simapp/template/.simapp/config/genesis.json:292]:
```suggestion
          "base": "cony",
          "display": "finschia"
```

**Review by 0Tech — APPROVED** [review]
LGTM, some nits.

**Review by jaeseung-bae — APPROVED** [review]

File: packages/finschia/src/modules/collection/queries.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import { createProtobufRpcClient, QueryClient } from "@cosmjs/stargate";
 import { assert } from "@cosmjs/utils";
-import { Any } from "cosmjs-types/google/protobuf/any";
 import { Coin, Contract, Grant, NFT, TokenType } from "@finschia/finschia-proto/lbm/collection/v1/collection";
 import { QueryClientImpl } from "@finschia/finschia-proto/lbm/collection/v1/query";
+import { Any } from "cosmjs-types/google/protobuf/any";
```
[the remark this answers is not in the document]
Comment (zemyblue) [reply packages/finschia/src/modules/collection/queries.ts:5]: I fixed lint error.

File: packages/finschia/src/modules/feegrant/queries.ts [hunk 4]
```diff
@@ -1,10 +1,10 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 import { createPagination, createProtobufRpcClient, QueryClient } from "@cosmjs/stargate";
-import { QueryAllowanceResponse, QueryAllowancesResponse } from "cosmjs-types/cosmos/feegrant/v1beta1/query";
 import {
   QueryAllowancesByGranterResponse,
   QueryClientImpl,
 } from "@finschia/finschia-proto/cosmos/feegrant/v1beta1/query";
+import { QueryAllowanceResponse, QueryAllowancesResponse } from "cosmjs-types/cosmos/feegrant/v1beta1/query";
```
[the remark this answers is not in the document]
Comment (zemyblue) [reply packages/finschia/src/modules/feegrant/queries.ts:7]: I fixed lint error. So I added #106 issue for fix this problem.

File: packages/finschia/src/modules/foundation/aminomessages.spec.ts [hunk 5]
```diff
@@ -643,6 +667,32 @@ describe("AminoTypes", () => {
       expect(aminoMsg).toEqual(expected);
     });
 
+    it("MsgUpdateCensorship", () => {
```
[the remark this answers is not in the document]
Comment (zemyblue) [reply packages/finschia/src/modules/foundation/aminomessages.spec.ts:670]: This is sub-test of `AminoTypes`. https://github.com/Finschia/finschia-js/blob/e6c0851b20a95104ca0b172e601b526f1866dc1b/packages/finschia/src/modules/foundation/aminomessages.spec.ts#L417

And, it is displayred like below when test.
스크린샷 2023-10-24 오후 1 28 54

So, the msg name was used in order to recognize the test result.

File: packages/finschia/src/modules/feegrant/queries.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (tkxkd0159) [reply packages/finschia/src/modules/feegrant/queries.ts:7]: Based on #106, It seems like there is a chance to find more points to need to lint in this repo. What about make another PR for #106 and rollback these unrelated changes in this PR to track our codes easily in the future?

[the remark this answers is not in the document]
Comment (zemyblue) [reply packages/finschia/src/modules/feegrant/queries.ts:7]: OK, that's good.

**Review by tkxkd0159 — APPROVED** [review]

**Review by 0Tech — APPROVED** [review]

**Pull request merged**